api/pom.xml

```diff
@@ -20,12 +20,12 @@
   <parent>
     <groupId>org.apache.livy</groupId>
     <artifactId>livy-main</artifactId>
-    <version>0.7.1.3.2.3.0-SNAPSHOT</version>
+    <version>0.7.1.3.2.3.2-3</version>
```

Comment:
This still looks wrong, it should be 0.7.1.3.2.3.0-SNAPSHOT.
Can you revert these changes?

Reply:
As per commits Release 3.2.3.2-2 and Release 3.2.3.2-3, 0.7.1.3.2.3.0-SNAPSHOT is getting changed